node-repository/src/main/java/com/yahoo/vespa/hosted/provision/maintenance/MetricsReporter.java

```diff
@@ -117,7 +127,32 @@ private void updateNodeMetrics(Node node) {
             // Ignore
         }
 
-        // TODO: Also add metric on whether some services are down on node?
+        long numberOfServices;
+        HostName hostName = new HostName(node.hostname());
+        List<ServiceInstance> services = servicesByHost.get(hostName);
+        if (services == null) {
+            numberOfServices = 0;
+        } else {
+            Map<ServiceStatus, Long> servicesCount = services.stream().collect(
+                    Collectors.groupingBy(ServiceInstance::serviceStatus, Collectors.counting()));
+
+            numberOfServices = servicesCount.values().stream().mapToLong(Long::longValue).sum();
+
+            metric.set(
+                    "numberOfServicesUp",
+                    servicesCount.getOrDefault(ServiceStatus.UP, 0L), context);
```

Comment:
Consider having each argument to `metric.set()` on separate lines or all on same lines. Now it is easy to mistake `context` as an argument to `getOrDefault`.

Reply:
Thanks, that's usually what I do. Fixed.